modules/installation-aws-upi-requirements.adoc

```diff
@@ -0,0 +1,399 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-aws-upi-requirements-{context}"]
+= Required AWS UPI infrastructure components
+
+To install {product-title} on user-provisioned infrastructure in
+Amazon Web Services (AWS), you must manually create both the machines and their
+supporting infrastructure.
+
+You can use the provided CloudFormation templates to create this infrastructure,
+or you can manually create the components. Review the CloudFormation templates
+for more details about how the components interrelate.
+
+[id="installation-aws-upi-cluster-machines-{context}"]
+== Cluster machines
+
+Create `AWS::EC2::Instance` objects for the following machines:
+
+* A bootstrap node. This machine is required during installation, but you
+can remove it after your cluster deploys.
+* At least three control plane machines. The control plane machines are not
+governed by a machineset.
+* Worker nodes. You can create these manually during the cluster installation
+process or you can create and control them by using a MachineSet after
+you create a bootstrap node.
+
+[id="installation-aws-upi-other-infrastructure-{context}"]
+== Other infrastructure components
+
+* A VPC
+* DNS entries
+* Load balancers and listeners
+* A Route53 zone
+* Security groups
+* IAM roles
+* S3 buckets
+
+.Required VPC components
+
+You need to create a VPC and subnets that allow communication to your machines.
```

Comment:
Isn't one of the UPI drivers that you can recycle existing infra?  You don't need to *create* a VPC, you just need to *have* one.  Can we rephrase this and other parts of this doc to make that distinction ("If you don't have an existing VPC, you can create one with our handy CloudFormation config or whatever you want to use instead...").